src/seedu/addressbook/parser/Parser.java

```diff
@@ -11,17 +11,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import seedu.addressbook.commands.AddCommand;
-import seedu.addressbook.commands.ClearCommand;
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```

Comment:
Again, do you see an issue with the import statement? 

Be careful because Intellij has a tendency to automatically change import statements into this, but this is a coding style violation!

Reply:
The reason I should not do that is because it clutters my local namespace.  Currently, IntelliJ always change it for me. How do I prevent this from happening? Or do I have to manually revert it back every time I see this?